client/js/recurring_donations/edit/index.es6

```diff
@@ -199,9 +197,9 @@ function view(state) {
         h('td.strong', 'Address')
       , h('td', [
           h('small', [
-            [supporter.address, supporter.city].filter(R.identity).join(', ')
+            [supporter.address, supporter.city].join(', ')
```

Comment:
What did `filter(R.identity)` do? What was trying to be achieved with it? Any ideas?

Reply:
Ahh I see, it is doing something, good catch. If you filter by the thing itself, it gets past the filter if it's truthy, but not if it's falsy. So you get things like
```js
[1, 2, false, 3, 0, 4, ''].filter(i => i)  // [1, 2, 3, 4]
```
so I should put this back

Reply:
(and the one just below)